docs/data-sources/archive_logs.md

```diff
@@ -9,24 +9,16 @@ Provides a Logz.io archive logs resource. This can be used to create and manage
 
 ## Attribute Reference
 * `storage_type` - (String) Specifies the storage provider.
-  If `S3`, the `amazon_s3_storage_settings` are relevant.
-  If `BLOB`, the `azure_blob_storage_settings` are relevant.
-* `amazon_s3_storage_settings` - (Object) Applicable settings when the `storage_type` is `S3`.
-* `azure_blob_storage_settings` - (Object) Applicable settings when the `storage_type` is `BLOB`.
 * `enabled` - (Boolean) If `true`, archiving is currently enabled.
 * `compressed` - (Boolean) If `true`, logs are compressed before they are archived.
-* `amazon_s3_storage_settings.credentials_type` - (String) Specifies which credentials will be used for authentication.
-* `amazon_s3_storage_settings.path` - (String) Specify a path to the **root** of the S3 bucket.
-* `amazon_s3_storage_settings.s3_secret_credentials` - (Object) Applicable settings when the `credentials_type` is `KEYS`.
-* `amazon_s3_storage_settings.s3_iam_credentials_arn` - (String) Amazon Resource Name (ARN) to uniquely identify the S3 bucket.
-* `amazon_s3_storage_settings.s3_external_id` - (String) The external id that gives Logz.io access to your S3 bucket.
-* `amazon_s3_storage_settings.s3_secret_credentials.access_key` - (String) AWS access key.
-* `amazon_s3_storage_settings.s3_secret_credentials.secret_key` - (String) AWS secret key.
-* `azure_blob_storage_settings.tenant_id` - (String) Azure Directory (tenant) ID. The Tenant ID of the AD app.
-* `azure_blob_storage_settings.client_id` - (String) Azure application (client) ID. The Client ID of the AD app.
-* `azure_blob_storage_settings.client_secret` - (String) Azure client secret.
-* `azure_blob_storage_settings.account_name` - (String) Azure Storage account name.
-* `azure_blob_storage_settings.container_name` - (String) Name of the container in the Storage account.
-* `azure_blob_storage_settingspath` - (String) Optional virtual sub-folder specifiying a path within the container.
+* `credentials_type` - (String) Applicable when `storage_type` is `S3`. Specifies which credentials will be used for authentication.
+* `s3_path` - (String) Applicable when `storage_type` is `S3`. Specify a path to the **root** of the S3 bucket.
+* `s3_iam_credentials_arn` - (String) Applicable when `storage_type` is `S3`. Amazon Resource Name (ARN) to uniquely identify the S3 bucket.
+* `aws_access_key` - (String) Applicable when `storage_type` is `S3`. AWS access key.
+* `tenant_id` - (String) Applicable when `storage_type` is `BLOB`. Azure Directory (tenant) ID. The Tenant ID of the AD app.
```

Comment:
Why we removed the prefix? i think we need to keep the prefix for the relevant cloud vendor

Reply:
The API changes made it impossible to keep in that structure. I had to make the change.